charon/tests/ui/unsupported/rename_attribute_failure.rs

```diff
@@ -0,0 +1,14 @@
+//@ known-failure
+//@ no-check-output
```

Comment:
We actually want to check the output, to make sure the error messages are correctly maintained